### PR TITLE
Improve usage of zookeeper cli

### PR DESCRIPTION
**Pull request by murong00**
### _Why are the changes needed?_
This change enabled the JLine support so that the `kyuubi-zookeeper-shell` command can be easier to use with history, prompt and completion, etc. Note that the `jline` version `0.9.94` is dependent on `zookeeper` version `3.4.14` which is a lower version. BTW, users can consider upgrading the `zookeeper` version for the sake of performance improvements and new features.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by pan3793** [comment]
sounds that jline is optional for zookeeper-shell?

**Comment by murong00** [comment]
> sounds that jline is optional for zookeeper-shell?

The `zkCli.sh` enabled this as default for flexibility and efficiency of use.

**Comment by pan3793** [comment]
The change make sense to me.
BTW, we keep using zk 3.4 client because of https://github.com/apache/incubator-kyuubi/issues/838#issuecomment-883241209

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
CI will fill, you need to update dependencyList, and a BSD license copy should be add into `license-binary/`

**Comment by pan3793** [comment]
@yaooqinn Does `kyuubi-zookeeper-shell` too long? Maybe we can short the name

**Comment by yaooqinn** [comment]
> @yaooqinn Does `kyuubi-zookeeper-shell` too long? Maybe we can short the name

I don't have a strong opinion here, what's your advice?

**Comment by pan3793** [comment]
The original script names `zkCli.sh`, I suggest we rename it to `zk-cli` to match the style of other kyuubi commands

**Comment by murong00** [comment]
> The original script names `zkCli.sh`, I suggest we rename it to `zk-cli` to match the style of other kyuubi commands

Seems ok to me, renamed to `kyuubi-zk-cli`. Thanks for the above tips.

**Comment by pan3793** [comment]
Thanks, merging to master

File: kyuubi-assembly/pom.xml [hunk 1]
```diff
@@ -94,6 +94,18 @@
...
+            <groupId>jline</groupId>
+            <artifactId>jline</artifactId>
+            <version>${jline.version}</version>
+            <exclusions>
+                <exclusion>
+                    <artifactId>junit</artifactId>
+                    <groupId>junit</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```
Comment (pan3793) [inline kyuubi-assembly/pom.xml:107]: Sorry, I just notice you pin `<version>` and `<exclusions>` directly in child pom `<dependencies>` block, could you please move them to root pom `<dependencyManagement>` block? @murong00

Comment (murong00) [reply kyuubi-assembly/pom.xml:107]: @pan3793 Sure, refined in #1416, please help to take a look, thanks.

**Pull request closed without merging**